modules/components/pages/outputs/redis_hash.adoc

```diff
@@ -353,6 +353,14 @@ This field supports xref:configuration:interpolation.adoc#bloblang-queries[inter
 
 *Default*: `{}`
 
+```yml
+# Examples
+
+fields:
+  foo: "${!this.foo}"
+  bar: "pew-${!this.bar.trim_prefix("sku-")}"
```

Comment:
I think you may want to use single quotes to avoid clashing with the inner double quotes:

```yaml
fields:
  foo: '${!this.foo}'
  bar: 'pew-${!this.bar.trim_prefix("sku-")}'
```

Reply:
Good catch!